src/Simplex/Messaging/Agent.hs

```diff
@@ -156,9 +161,7 @@ processCommand c@AgentClient {sndQ} st (corrId, connAlias, cmd) =
       withStore (getConn st cAlias) >>= \case
         SomeConn _ (DuplexConnection _ rq _) -> subscribe rq
         SomeConn _ (RcvConnection _ rq) -> subscribe rq
-        -- TODO possibly there should be a separate error type trying
-        -- TODO to send the message to the connection without RcvQueue
-        _ -> throwError PROHIBITED
+        _ -> throwError $ CONN SIMPLEX
```

Comment:
I'm not sure SIMPLEX is descriptive enough.. RcvConnection is also simplex. Maybe `CONN ONLY_RCV`? Just a thought, not sure it's necessary to change